src/table/utils/handle-click.ts

```diff
@@ -57,10 +57,12 @@ export const getSelectionMouseHandlers = (
   };
 
   const handleMouseUp = (evt: React.MouseEvent) => {
-    if (areBasicFeaturesEnabled && evt.button === 0) {
-      selectionDispatch({ type: SelectionActions.SELECT_MOUSE_UP, payload: { cell, evt, announce } });
-    } else if (!areBasicFeaturesEnabled && cell.isSelectable) {
-      selectionDispatch({ type: SelectionActions.SELECT, payload: { cell, evt, announce } });
+    if (evt.button === 0) {
```

Comment:
complexity goes up a bit for not early returning here, but I think it is no problem here +  readability is better